2019/June/README.md

```diff
@@ -1 +1,34 @@
-# talk links and description
+# June Python Pune meetup 2019
+
+## Logging in Python
+  * **Speaker**: Akshay Gaikwad
+  * **Description**: In simple term, logging is a way to understand
+    what is going on inside an application. It can help programmer to
+    understand flow of a program. With logging, it is easy to
+    constantly look at the flow that an application going
+    through. Python has simple and powerful logging module as standard
+    library from 2.3, which described in PEP-282.
+
+  * [Slides](https://gitlab.com/akshay196/slides/blob/master/logging/presentation.org)
```

Comment:
How about adding source here as well? That will give more clarity.

Reply:
Done

Reply:
Done